### ISSUE TITLE
Display whitelisted users in the frontend

### ISSUE THREAD

**Issue by d89** [h-da/geli#190]
I want to see, which users from the CSV-upload are actually whitelisted and which of those whitelisted users have signed up in the meantime.

**Comment by d89** [comment]
Somehow the things we discussed together are not present?

* The interface to add students to the whitelist is still outside the box:
![image](https://user-images.githubusercontent.com/423154/35198572-27fea5c8-fef1-11e7-9a18-cba52f41b607.png)
* The editing-pen is grayed out, didn't we want to remove it when it is not clickable?

Another thing I noticed: When I upload a fully invalid csv file like the following, It is not telling me about any problems, even though no students could be imported. What I'd like to have besides the discussed warning that all the students from the (previous) whitelist get overridden is some kind of summary, what has happened - e.g. "Imported 3 students, could not import 2 students (lines 14, 15)" or the like.

```
Teilnehmerliste 21.01.2018 21:04

DIESE TEILNEHMERLISTE DARF NICHT ZUR MELDUNG VON NOTEN ODER TESTATEN VERWENDET WERDEN !

[30.2566]
WS1718 
Müller,D. / Dörge / Sauer,T.
Fortgeschrittene Webentwicklung

"// Datenschutz: Namen und Matrikelnummern dürfen nicht zusammen veröffentlicht werden ! "

xx
;;;Müller;Studentendavid;1234123;Bachelor 7C;Di5 Mi1;
;;;Müller;Willneverlogin;1234129;Bachelor 7C;Di5 Mi1;
yy
```
